hearpreprocess/pipeline.py

```diff
@@ -798,6 +867,19 @@ def run(self):
                 indent=True,
             )
 
+            # Save the slug and the label in as the split metadata
+            audiolabel_df.to_csv(
+                self.workdir.joinpath(f"{split_path.stem}.csv"),
+                index=False,
+            )
+
+            split_label_count[f"{split_path.stem}"] += audiolabel_df["label"].nunique()
+
+        diagnostics.info(
+            f"{self.longname} - Unique labels in each split: "
```

Comment:
Again. Why unique labels?

Count the number of labels and show their percentage.

Also, show a count for labels that are MISSING.

And order them by percentage.

That is much more informative than looking at a list of labels in some random order. Are you really going to spot a bug like that?